hydro_deploy/core/src/hydroflow_crate/service.rs

```diff
@@ -287,7 +287,10 @@ impl Service for HydroflowCrateService {
             .unwrap();
 
         let start_ack_line = ProgressTracker::leaf(
-            "waiting for ack start".to_string(),
+            self.display_id
```

Comment:
Can we format this as `SERVICE NAME / waiting for ack start` to match the usual `ProgressTracker` style?